main.rb

```diff
@@ -0,0 +1,25 @@
+require_relative './student'
+require_relative './teacher'
+require_relative './classroom'
+require_relative './book'
+require_relative './rental'
+require_relative './inputs'
+require_relative './library'
+
+class Run
+  include Inputs
+  def main
+    library = Library.new
+    puts 'Welcome to School Library App'
+    flag = true
+    while flag
+      library.display_menu
+      option = input_number(7)
+      flag = false if option == 7
+      library.display_choice(option)
+    end
```

Comment:
- [ ] [OPTIONAL] Great job giving the app an exit option, however, for a better user experience (UX) you can `print "Thank you for using this app!"` when the user exits the app.